lib/pg/query/delete.js

```diff
@@ -1,28 +1,12 @@
-/**
- * Handle delete calls to postgres
+const { processCQN } = require('../execute')
+
+/*/**
```

Comment:
would you mind fixing the JSDoc here
/** -> start
dbc is of type PoolClient
cqn probably String|Object?